dynawo/sources/Modeler/DataInterface/DYNVoltageLevelInterfaceIIDM.cpp

```diff
@@ -345,9 +345,6 @@ VoltageLevelInterfaceIIDM::calculateBusTopology() {
         calculatedBus_[component[*iter]]->setAngle0(itBBS->angle());
     }
   }
-
-  for (unsigned int i = 0; i < calculatedBus_.size(); ++i)
-    Trace::debug("NETWORK") << (*calculatedBus_[i]) << Trace::endline;
```

Comment:
This one gives the details of the calculated buses (nodes and bus bar sections), I am not sure whether to keep it or delete it

Reply:
I remember that it has been useful at the time where we implemented the voltage level, the connection/disconnection propagation and the graph algorithms but I haven't looked at it since a long time...

Reply:
To decide, I have done a very simple test: I opened the log for one test case and looked if it took me more than five minutes to understand the log, its meaning and the way I can use it. 
The test is positive: I find the log comprehensive and topological issues are always a nightware to debug. 
Thus I would suggest to keep it.

Reply:
Done